### ISSUE TITLE
Javascript alternitive?

### ISSUE THREAD

**Issue by hg0428** [JSBSim-Team/jsbsim#980]
I saw JS in the name and hoped that this was JS, but then I clicked on it and it wasn't.
Is there any way I could integrate this with THREE.js via WASM?

**Comment by seanmcleod** [comment]
Nope, they're part of the initials of the person who started the project :wink:.

I don't see any particular reason you couldn't compile JSBSim to WASM, although I'm not aware of anyone trying to do so.

**Comment by hg0428** [comment]
It would be hard to get this to integrate with our current code, even via wasm. I guess I need a different solution. If anyone has ideas, please tell me.
It's looking like I will have to build my own.
